ruma-common/src/push.rs

```diff
@@ -8,6 +8,7 @@ use serde::{Deserialize, Serialize};
 mod action;
 mod any_push_rule;
 mod condition;
+mod predefined_push_rules;
```

Comment:
That's quite the long module name, I think `predefined` would suffice (or maybe `server_defaults` or sth like that).